### DISCUSSION TITLE
What is the difference between .md and .mdoc

### DISCUSSION THREAD

**Discussion by fahmiidris** [markdoc/markdoc#352] [General]
I'm trying to learn markdoc and trying to apply it to nextjs with the next.js plugin provided, but I'm confused about the difference between the two extensions (`.md` and `.mdoc`). I can't find it in the documentation.

Can anyone help explain it to me? GPT Chat also doesn't have any information about this hehe.

**Comment by nkohari** [comment]
There's actually no difference. If you want to support both Markdoc and Markdown articles in your project, you can use `.mdoc` for Markdoc and `.md` for Markdown, but it's entirely up to you. Sorry that it's not clear in the documentation; I'll see how we can improve that!
